classes/class-paysoncheckout-for-woocommerce-order-management.php

```diff
@@ -204,93 +204,76 @@ class_exists( 'WC_Subscriptions' )
 	 * @return boolean Did the refund go through okay?
 	 */
 	public function refund_payment( $order_id ) {
-				$query_args = array(
-					'fields'         => 'id=>parent',
-					'post_type'      => 'shop_order_refund',
-					'post_status'    => 'any',
-					'posts_per_page' => -1,
-				);
-
-				$refunds = get_posts( $query_args );
-
-				$refund_order_id = array_search( $order_id, $refunds );
-				if ( is_array( $refund_order_id ) ) {
-					foreach ( $refund_order_id as $key => $value ) {
-						$refund_order_id = $value;
-						break;
-					}
-				}
+		$order        = wc_get_order( $order_id );
+		$refund_order = $order->get_refunds()[0];
```

Comment:
I know this is not unique to this change in the code. But what happens here when we have multiple refunds? Will the newest always be the first index?

Reply:
Yes, the newest (the refund we're currently processing) is always at index 0, and the previous refunds are "shifted" to the next index.